igvm/migratevm.py

```diff
@@ -2,126 +2,9 @@
 
 Copyright (c) 2018, InnoGames GmbH
 """
+# XXX: This module is for backwards compatibility.
```

Comment:
Where do we use this still ?

Reply:
Any script that needs it.  This is the current place of the function.